README.md

```diff
@@ -124,5 +124,9 @@ Because we are now treating the ARM Template as an IL, we expect and encourage o
 * [Farmer](https://compositionalit.github.io/farmer/) (@isaacabraham) - Generate and deploy ARM Templates on .NET
 * [Cloud Maker](https://cloudmaker.ai) (@cloud-maker-ai) - Draw deployable infrastructure diagrams that are converted to ARM templates
 
+## Telemetry
+
+VS Code collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=528096&clcid=0x409) to learn more. If you don’t wish to send usage data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false`. Learn more in our [FAQ](https://code.visualstudio.com/docs/supporting/faq#_how-to-disable-telemetry-reporting).
```

Comment:
This should probably be more specific about vscode vs others (CLI, MSBuild/etc).  I'm assuming the teleemtry goes nowhere if not running in the vscode extension, right?

Reply:
This was taken directly from ARM Tools.

Reply:
We use the same in our other extensions e.g. Azure Functions https://github.com/Microsoft/vscode-azurefunctions#telemetry

Reply:
Yes, but this readme is for all of Bicep, including CLI.  Not just vscode.  I'd add something like "When using the Bicep extension in VS Code, ..."

We want people to understand this only applies to vs code and how to turn it off.  Some are very sensitive about telemetry.

Reply:
It clearly states VS code in the message. Not sure how we could make that more clear.

Reply:
I just told you how. :-)

Reply:
The message starts with VS code. I think it's pretty clear. I'd like to align with the other extensions as much as possible.